waspc/data/Generator/templates/server/src/auth/user.ts

```diff
@@ -0,0 +1,15 @@
+import type { SanitizedUser as User } from '../_types/index' 
```

Comment:
We can't import server code from frontend (we can import types) but these are functions.

We can't put the frontend and the backend code into the `universal` folder since they both rely on importing types, but for the server and the client it would be a different import path.

Instead of templating the types import, I went with simple duplication.

Reply:
Hm ok. You have a better feeling for there then I do, so I won't push too much, but it is not trivial amount of code + I think it will get a bit more complex with the suggested changes. Maybe it would be worth somehow un-duplicating it. I would at least put some comments in both files to let reader know that same file exists somewhere else and that changes they are doing should likely also be done there.

Reply:
I can put a comment in both files 👍